README.md

```diff
@@ -11,4 +13,10 @@ docker run --rm --name curator \
   -e SSL=True \
   meteogroup/curator:latest
 ```
-
+List of environment settings:
+- ELASTICSEARCH_HOST - endpoint address to connect to Elastisearch cluster, default "localhost"
+- ELASTICSEARCH_PORT - endpoint port to connect to cluster, default 9200
+- SSL - if access to Elasticsearch is protected by SSL, has to be "True", default "False"
+- INDEX_PREFIX - name for the prefix, default "dataprovisioning*"
+- TIME_STRING - default '%Y.%m.%d'
+- DELETE_DAY - default 14
```

Comment:
A bit off-topic for this PR, but still:
What do you think of copying everything to /opt/curator or /var/curator ... or anywhere else but /tmp in the build?
Would make more sense to me

Reply:
yeah, I would go with /var/curator as binary can be found by usual path